examples/twentyseven-pt-stencil-solver/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_executable(twentyseven - pt - stencil - solver twentyseven - pt - stencil -
```

Comment:
The same comment as the previous here for formatting

Reply:
fixed this. I used clang-format incorrectly. Using `make format` now.